release-tests/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "@noir-lang/release-tests",
+  "version": "0.0.0",
+  "main": "index.js",
+  "license": "MIT",
```

Comment:
```suggestion
```
Might as well remove this field, yeah?

Reply:
It avoid a warning and there's no harm.

Reply:
If you want to keep it, I think it should match noir's license and be the SPDX for "Apache or MIT" so as not confuse anyone looking around the project.